packages/react-examples/src/react/Theme/docs/ThemesOverview.md

```diff
@@ -1,69 +1 @@
-The entire color palette of the controls is themeable. We provide a set of sensible defaults, but you can override all colors individually. To do this, you must call `loadTheme()` with a custom theme object at app startup.
-
-### Using themes
-
-To use themes, an application must call `loadTheme()` immediately at app startup before any app code executes.
-Here is an example:
-
-```tsx
-import { loadTheme } from '@fluentui/react';
-
-loadTheme({
-  palette: {
-    themePrimary: '#0078d4',
-    themeLighterAlt: '#eff6fc',
-    themeLighter: '#deecf9',
-    themeLight: '#c7e0f4',
-    themeTertiary: '#71afe5',
-    themeSecondary: '#2b88d8',
-    themeDarkAlt: '#106ebe',
-    themeDark: '#005a9e',
-    themeDarker: '#004578',
-    neutralLighterAlt: '#f8f8f8',
-    neutralLighter: '#f4f4f4',
-    neutralLight: '#eaeaea',
-    neutralQuaternaryAlt: '#dadada',
-    neutralQuaternary: '#d0d0d0',
-    neutralTertiaryAlt: '#c8c8c8',
-    neutralTertiary: '#c2c2c2',
-    neutralSecondary: '#858585',
-    neutralPrimaryAlt: '#4b4b4b',
-    neutralPrimary: '#333333',
-    neutralDark: '#272727',
-    black: '#1d1d1d',
-    white: '#ffffff',
-  },
-});
-```
-
-### Customization
-
-You can also add some of your own customizations to the default theme.
-For example, you can set the `defaultFontStyle` property to modify every font variant
-(small, medium, large, etc.), or you can target specific variants through the `fonts` property.
-These can be used separately, or together, as shown in the example below.
-The overrides can include any property from [`IRawStyle`](#/controls/web/references/irawstyle).
-
-```tsx
-import { loadTheme } from '@fluentui/react';
-
-loadTheme({
-  defaultFontStyle: { fontFamily: 'Monaco, Menlo, Consolas', fontWeight: 'regular' },
-  fonts: {
-    small: {
-      fontSize: '11px',
-    },
-    medium: {
-      fontSize: '13px',
-    },
-    large: {
-      fontSize: '20px',
-      fontWeight: 'semibold',
-    },
-    xLarge: {
-      fontSize: '22px',
-      fontWeight: 'semibold',
-    },
-  },
-});
-```
+The entire color palette of the controls is theme-able. We provide a set of sensible defaults, but you can override all colors individually. To do this, you can reference [this wiki](https://github.com/microsoft/fluentui/wiki/How-to-apply-theme-to-Fluent-UI-React-components) for more details.
```

Comment:
Would you mind also adding a link to the font customization docs?

Reply:
Updated. I don't want to specifically callout "here is how to override default fonts" because it's too specific of a case. I added the links to the color/typography pages in a more generic way. let me know what you think

Reply:
People wanting to override the default fonts (mainly to get around issues with the assets license, or various on-prem scenarios) is more common than you might think, which is why we added docs on it. But I can see how it might be better to stick with more generic information.